functions/routes/ecom/modules/create-transaction.js

```diff
@@ -81,7 +81,7 @@ exports.post = ({ appSdk, admin }, req, res) => {
     vindiBill.installments = installmentsNumber
   } else {
     // banking billet
-    vindiBill.payment_method_code = 'bank_slip'
+    vindiBill.payment_method_code = config.banking_billet.bankSlipType ? config.banking_billet.bankSlipType : 'bank_slip'
```

Comment:
Na sua implementação deveria ser um OR aqui em vez de ternário:

```js
    vindiBill.payment_method_code = config.banking_billet.bankSlipType || 'bank_slip'
```

Mas acho que é mais simples pro usuário só com o booleano, então:

```suggestion
    vindiBill.payment_method_code = config.banking_billet.is_yapay ? 'bank_slip_yapay' : 'bank_slip'
```